src/Store/StoreThread.re

```diff
@@ -80,13 +81,8 @@ let start =
       cliOptions,
     );
 
-  let state = Model.State.create();
-
-  let latestState: ref(Model.State.t) = ref(state);
   let latestRunEffects: ref(option(unit => unit)) = ref(None);
 
-  let getState = () => latestState^;
-
```

Comment:
Instead of storing the state here as well, the `getState` function is just passed in from `Oni2_editor` instead.